app/views/reports/standard_form_report/subset/_full_width_summary_row.html.erb

```diff
@@ -0,0 +1,27 @@
+<tr class="summary full_width_subhead">
+  <td class="num">#<%= summary.questioning.full_dotted_rank %></td>
+  <td colspan="<%= result_cols + 2 %>">
+    <%= render("reports/standard_form_report/form_summary_question_label", summary: summary) %>
+  </td>
+</tr>
+
+<%# if no items, show the 'no data' message %>
```

Comment:
The comments below here can go I think, and then you can collapse the blank spaces.

Reply:
done